evm_loader/lib/src/config.rs

```diff
@@ -62,7 +64,15 @@ pub fn load_api_config_from_environment() -> APIOptions {
         .and_then(|v| Pubkey::from_str(&v).ok())
         .expect("SOLANA_KEY_FOR_CONFIG variable must be a valid pubkey");
 
-    let db_config = load_db_config_from_environment();
+    let tracer_db_type = env::var("TRACER_DB_TYPE")
+        .map(|db_type| match db_type.to_lowercase().as_str() {
+            "clickhouse" => Ok("clickhouse"),
+            "rocksdb" => Ok("rocksdb"),
+            _ => Err("Invalid tracer_db type!"),
+        })
+        .unwrap()
+        .expect("TRACER_DB_TYPE variable must be either 'clickhouse' or 'rocksdb'")
+        .to_string();
```

Comment:
I saw this code several times. I think it should be implemented once.
Another question, why do you use `TRACER_DB_TYPE`? The other approach is to connect to clickhouse if NEON_DB_CLICKHOUSE_URLS is set. Otherwise parse ROCKSDB_HOST and connect to RocksDB.
And maybe rename ROCKSDB_HOST to NEON_DB_ROCKS_URL?